### PR TITLE
ccip: update contract reader config.

### PR DESCRIPTION
**Pull request by winder**
Add configuration for the new getAllSourceChainConfig function.

### CODE REVIEW

File: core/capabilities/ccip/configs/evm/contract_reader.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ var DestReaderConfig = evmrelaytypes.ChainReaderConfig{
 					ChainSpecificName: mustGetMethodName("getSourceChainConfig", offrampABI),
 					ReadType:          evmrelaytypes.Method,
 				},
+				"OffRampGetAllSourceChainConfigs": {
```
Comment (0xAustinWang) [inline core/capabilities/ccip/configs/evm/contract_reader.go:86]: let's add this to consts to maintain the style in the rest of the file

Comment (winder) [reply core/capabilities/ccip/configs/evm/contract_reader.go:86]:
```suggestion
				// TODO: replace with const when chainlink-ccip is updated
				"OffRampGetAllSourceChainConfigs": {
```

Comment (winder) [reply core/capabilities/ccip/configs/evm/contract_reader.go:86]: I was tempted to make a chainlink-ccip PR just to add the const, but there are some pending multi-repo merges that make updating the dependency difficult.

**Review by asoliman92 — APPROVED** [review]

File: core/capabilities/ccip/configs/evm/contract_reader.go (hunk 1 above)
Comment (makramkd) [reply core/capabilities/ccip/configs/evm/contract_reader.go:86]: TODO is fine for now

**Review by makramkd — APPROVED** [review]

**Pull request merged**